camera_lucidTypes.hpp

```diff
@@ -209,8 +261,18 @@ namespace camera_lucid {
             base::samples::frame::frame_mode_t::MODE_BAYER_RGGB;
         /** Depth*/
         uint8_t depth = 8;
+        /** Short Exposure State
+         *  This state needs a exposure_time between 2.464 and 1 microseconds
+         *  and is not compatible with Exposure_Auto_Limit_Auto_OFF when in
+         *  EXPOSURE_AUTO_CONTINUOUS. */
+        bool short_exposure_enable = false;
         /** Sets the automatic exposure mode.*/
         ExposureAuto exposure_auto = EXPOSURE_AUTO_CONTINUOUS;
+        /** Sets the automatic exposure algorithm*/
+        ExposureAutoAlgorithm exposure_auto_algorithm = EXPOSURE_AUTO_ALGORITHM_MEAN;
+        /** Sets the automatic exposure damping represented as %.
+         * Maximum, Minimum: 99.6094, 0.390625 */
+        double exposure_auto_damping = 89.8438;
```

Comment:
I think that it's probably camera-dependent. I'd prefer that the default is `base::unknown<double>()` (i.e. NaN) and that you do not set the value if it is nan (plus update comment)